src/main/java/net/atos/zac/flowable/cmmn/CMMNService.java

```diff
@@ -1,23 +1,23 @@
 /*
- * SPDX-FileCopyrightText: 2022 Atos
+ * SPDX-FileCopyrightText: 2024 Lifely
```

Comment:
Should this be `SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely` ?

Reply:
oei!

Reply:
👍🏽 I noticed more cases in this PR BTW; you want me to make edits?

Reply:
fix them now hopefully!